pkg/common/scheduler/state/state.go

```diff
@@ -0,0 +1,299 @@
+/*
```

Comment:
@lionelvillard State file somehow got screwed up when moving to a new location and now shows up as a new file. So lost git history. Tried a few things but I think the ultimate fix might be a new branch and start from scratch to retrieve history....

Reply:
no biggie.